Rework README to reflect the maturing project

dhimmel opened the pull request:
This a major update to the README to be more informative than currently.

Requesting review from @cgreene and @KarinSpiderwoman.

@bvasko and @bdolly: I listed you two as frontend maintainers although you don't actually have write access yet... was hoping you could [introduce yourself](https://github.com/cognoma/cognoma/issues/2) first.

dhimmel committed:
Rework README to reflect the maturing project

dhimmel committed:
Add links to GitHub profiles

gwaybio commented on README.md:
```diff
@@ -2,50 +2,46 @@
-On Tuesday September 6, we're [meeting up](https://www.meetup.com/DataPhilly/events/233403039/ "DataPhilly Meetup Details") at CandiDate (Floor 17 of [230 S Broad Street](https://goo.gl/maps/Anoo4SUE9At "Industrious on Google Maps")) from 6:00–8:30 PM.
+| Team Name | Repositories | Description |
+|-----------|--------------|-------------|
+| Cancer Data | [`cancer-data`](https://github.com/cognoma/cancer-data) | processing the underlying cancer data to the formats required for this project. See . |
```
looks like something was left off the end of this description. "See ."

gwaybio commented on README.md:
```diff
@@ -2,50 +2,46 @@
+We hold project meetups. Our usual meeting spot is at Industrious (where CandiDate is located). The address is [230 S Broad St, Floor 17, Philadelphia](https://goo.gl/maps/Anoo4SUE9At "Google Maps").
 
-### July 26
+| 📅 Date | ⌚ Time | 🗺 Location | ℹ️ Meetup Details | 💰 Sponsor |
```
Is there any reason why you don't persist the hyperlinks for CandiDate, IBI, or Milkboy?

cgreene replied:
I agree that it would be nice to persist the hyperlinks for hosts and sponsors.

gwaybio commented on README.md:
```diff
@@ -2,50 +2,46 @@
+## Maintainers
 
-### July 19
+Cognoma relies on our generous community maintainers. Thanks to the following maintainers for their help:
```
Can you define what a "maintainer" is here?

cgreene replied:
Also, it may be helpful to note in text that the number of contributors is far larger than this. If it's possible to track who has participated outside the sandbox and automatically or periodically update a list of names here (github linked to usernames) of all contributors that would be great.

gwaybio commented on README.md:
```diff
@@ -2,50 +2,46 @@
++ **Backend**: Andrew Madonna ([@**awm33**](https://github.com/awm33)), Derek Goss ([@**dcgoss**](https://github.com/dcgoss)), Alan Elkner ([@**aelkner**](https://github.com/aelkner))
++ **Frontend**: Bonnie DiPasquale ([@**bvasko**](https://github.com/bvasko)), Benjamin Dolly ([@**bdolly**](https://github.com/bdolly))
++ **Community**: Karin Wolok ([@**KarinSpiderwoman**](https://github.com/KarinSpiderwoman))
++ **Wildcards**: Gregory Way ([@**gwaygenomics**](https://github.com/gwaygenomics)), Casey Greene ([@**cgreene**](https://github.com/cgreene)), Daniel Himmelstein ([@**dhimmel**](https://github.com/dhimmel))
```
yes! love the title. Makes me think of [Charlie Kelly](https://www.youtube.com/watch?v=MYtjpIwamos) (possibly NSFW (language) ?)

gwaybio reviewed (changes requested):
Really like the new format - there are only a couple fixes suggested

cgreene reviewed (commented):
I agree with @gwaygenomics suggestions. I'd love to see a full list of contributors but maybe this is something that is hard to put together.

dhimmel committed:
Address review comments

dhimmel commented:
@gwaygenomics & @cgreene I addressed some of your comments. I wasn't able to find an easy resource for showing all contributions across the organization. Perhaps a future pull request can implement a continuous analysis and update of a contribution document. Is deferring okay? -- I'd like to get this PR merged since it has the meetup details for next Monday.

cgreene reviewed (approved):
LGTM
